tests/src/Common/Platform/platformdefines.h

```diff
@@ -55,6 +55,11 @@ typedef const WCHAR *LPCWSTR, *PCWSTR;
 #define __stdcall
 #define _cdecl
 #endif
+#ifndef CROSS_MSC_VER_DEFINES
+#define CROSS_MSC_VER_DEFINES
+#define __FUNCSIG__ "<clang-3.5 signature not supported>"
+#define __int64 long long
+#endif // CROSS_MSC_VER_DEFINES
```

Comment:
@janvorli Could you tell me, which version of clang has been used, please? I am using clang-3.5 as documentation says.